### DISCUSSION TITLE
v1.2.4

### DISCUSSION THREAD

**Discussion by SylarLong** [SylarLong/iztro#50] [Announcements]
## v1.2.4

- 🪄 功能（feature）

  🇨🇳

  - 新增越南语支持

  🇺🇸

  - support Vietnamese

- 🛠️ 修复（fix）

  🇨🇳

  - 修复代码风格检查发现的额问题 #42 #43

  🇺🇸

  - fix codestyle issues #42 #43

<hr /><em>This discussion was created from the release <a href='https://github.com/SylarLong/iztro/releases/tag/v1.2.4'>v1.2.4</a>.</em>

**Comment by whoCan1** [comment]
horoscope 这个方法，1.2.4与1.2.3 想比1.2.2之前数据不一样，请问这个是什么问题

**Reply by SylarLong** [reply]
~hello，可以列出以下差异么？horoscope方法返回的数据结构是没有修改的呢~

**Reply by SylarLong** [reply]
请使用v1.2.3以后的版本哦，之前的版本在horoscope的索引计算上有错误，请参照 https://github.com/SylarLong/iztro/issues/36

**Comment by thiepwong** [comment]
好的。我先看看在答复你哦。今天忙了点

Vào Th 2, 18 thg 9, 2023 lúc 15:01 whoCan ***@***.***> đã
viết:

**Comment by whoCan1** [comment]
你好，我想问下你文档排盘的这个数据，怎么和horoscope返回的数据有差异啊
![image](https://github.com/SylarLong/iztro/assets/9334705/7ff33846-6b95-46da-a29f-a097da71f78e)

**Reply by SylarLong** [reply]
抱歉，目前demo里使用的还是老的代码，我会尽快更新demo的，请以 horoscope() 方法返回的数据为准

**Reply by whoCan1** [reply]
我想问下，在排盘里面定位，是根据horoscope()返回的index吗
![image](https://github.com/SylarLong/iztro/assets/9334705/cbb55715-90e1-49b5-9182-20119b21d65b)

**Reply by SylarLong** [reply]
是的

**Reply by whoCan1** [reply]
好的，非常感谢

**Reply by SylarLong** [reply]
不客气~有问题随时回来提问。

happy coding~

**Comment by whoCan1** [comment]
但是很奇怪的点是，我看你这个排盘和别的排盘数据又对的上，但是用你们的组件，调试数据又不一样，可不可以教我一下这个大限流年流月是怎么去定位的，谢谢

**Reply by SylarLong** [reply]
如果想了解`安星诀`的知识，可以查看 [紫微斗数安星诀](https://docs.iztro.com/learn/setup.html) 的文档，目前还没有更新完，可以持续关注哦。
